php/libraries/NDB_Client.class.inc

```diff
@@ -89,7 +89,7 @@ class NDB_Client
         if (!$this->_isWebClient) {
             // Set user as unix username.
             // Requires Loris to have user with this username
-            $user =& User::singleton(get_current_user());
+            User::singleton(getenv('USER'));
```

Comment:
What is the purpose of this line? Can we remove it? Is it to check if the user exists?

Reply:
@xlecours pretty sure thats to allow the DB to trackchanges when you run backend scripts in the history tables. it uses the unix user as lorisuser.

its annoying because it requires you to have a loris user with the same userid as ur unix user. and when it comes to 
`get_current_user` it was actually checking the owner of the PHP file rather than the actual logged in unix user..

I'm not 100% on it but @driusan should be able to tell if its true or not.

Reply:
@ridz My point is that trackchanges() already do that gymnastic and there is no point to do that here. Unless we remove it from trackchanges()